docs/roadmap.md

```diff
@@ -85,6 +85,14 @@ Exec into the Kubernetes Pod right from Argo CD Web UI! [#4351](https://github.c
 
 The feature allows combining of-the-shelf Helm chart and value file in Git repository ([#2789](https://github.com/argoproj/argo-cd/issues/2789))
```

Comment:
Could we merge this under `Support multiple sources for an Application`?

Reply:
I think it's fine to keep two items. People probably think of them as separate features.